docker-hub/api/dvp.yaml

```
@@ -6,7 +6,36 @@ info:
     url: https://docs.docker.com/images/logo-docker-main.png
     href: /reference
   description: |
-     The Docker DVP Data API allows [Docker Verified Publishers](https://docs.docker.com/docker-hub/publish/) to view image pull analytics data for their namespaces
+     The Docker DVP Data API allows [Docker Verified Publishers](https://docs.docker.com/docker-hub/publish/) to view image pull analytics data for their namespaces  
+       
+     #### Summary data
+
+     In your summary data CSV, you will have access to the data points listed below. You can request summary data for a complete week (Monday to Monday) or for a complete month (available on the 1st day of the following month). 
+
+     - For every namespace, repo, and reference (tag or digest):
+       - Unique IP address count
+       - Pulls by tag count
+       - Pulls by digest count
+       - Version Check count
+
+     #### Raw data
+
+     In your raw data CSV you will have access to the data points listed below. You can request raw data for a complete week (Monday to Monday) or for a complete month (available on the 1st day of the following month). Please note that each Action is represented as a single row.
```

I would not capitalize "Action"

```suggestion
     In your raw data CSV you will have access to the data points listed below. You can request raw data for a complete week (Monday through Sunday) or for a complete month (available on the first day of the following month). **Note:** each action is represented as a single row.
```